cmd/create/operatorroles/cmd.go

```diff
@@ -246,20 +263,27 @@ func run(cmd *cobra.Command, _ []string) {
 			os.Exit(1)
 		}
 	}
+	roleARN := cluster.AWS().STS().RoleARN()
+	roleName := strings.Split(roleARN, "/")
+	rosaPrefix := ""
+	if len(roleName) > 1 {
+		rosaPrefix = strings.Split(roleName[1], "-Installer-Role")[0]
+	}
```

Comment:
Doing this means that a whole chunk of code can be removed below (github doesn't let me comment in it, since it's hidden)
```
		policyARN := ""
		if prefix == "" {
			policyARN, err = awsClient.FindPolicyARN(operator, version)
			if err != nil {
				return err
			}
		}
		if policyARN == "" {
			policyARN = getPolicyARN(accountID, prefix, operator.Namespace, operator.Name)
}
```
it then becomes simply
```
		policyARN = getPolicyARN(accountID, prefix, operator.Namespace, operator.Name)
```
since there's _always_ a prefix

Reply:
> Ah, I see what you're doing - extracting the prefix from the cluster role to determine what the policy would be named. Interesting, I'll have to think about this - sounds like there could be some gotchas there

Yes I understand what you are saying. But unless we persist these prefix somewhere there is no way for us to associate it with the cluster. I was thinking to persist in the DB again but it would be the same kind of flow to determine the prefix user used while creating these policies

Reply:
No you're right, the prefix should always be the same as the one using during `create account-roles`. We don't need to support undocumented processes in the CLI. So this approach is fine.